packages/teams-js/src/public/video.ts

```diff
@@ -216,4 +216,153 @@ export namespace video {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.video ? true : false;
   }
+
+  /**
+   * @beta
+   * Namespace to get video frames from a media stream.
+   * When the host supports this capability, developer should call {@link mediaStream.registerForVideoFrame} to get the video frames instead of {@link registerForVideoFrame} to get the video frames, callback of {@link registerforvideoFrame} will be ignored when the host supports this capability.
+   */
+  export namespace mediaStream {
+    /**
+     * @beta
+     * Checks if video.mediaStream capability is supported by the host
+     * @returns boolean to represent whether the video.medisStream capability is supported
+     *
+     * @throws Error if {@linkcode app.initialize} has not successfully completed
+     *
+     */
+    export function isSupported(): boolean {
+      return ensureInitialized(runtime) && textureStreamAvailable() && !!runtime.supports.video?.mediaStream;
+    }
+
+    function textureStreamAvailable(): boolean {
```

Comment:
>textureStreamAvailable [](http://example.com/codeflow?start=13&length=22)

Consider prefacing the function name with `is` to make it clearer, e.g. `isTextureStreamAvailable()`

Reply:
updated